cluster/saltbase/salt/kubelet/default

```diff
@@ -121,5 +121,10 @@
   {% set network_plugin = "--network-plugin=opencontrail" %}
 {% endif -%}
 
+{% set kubelet_port = "10250" -%}
```

Comment:
This looks broken. If the pillar isn't defined, this won't pass `--port=10250` but will just pass `10250` on the command line. It'd be better to make the default just be an empty string so that we pick up the default value compiled in (with a shorter command line) if a different port isn't explicitly set.